rpa-builder/6.1/modules/ROOT/pages/_partials/toolbox-web-automation-web-session.adoc

```diff
@@ -98,26 +98,27 @@ closing the session at a specific point in the Workflow.
 //tag::wizardCommon[]
 == Wizard
 
-image:toolbox-variable-web-automation-web-session-images/image1.png[An image of the web session wizard window]
+The Web Session Wizard contains the *Reuse Web Session*, *General Settings*, and *Browser Specific Settings* sections.
 
-The *Reuse Web Session* section defines which web session is reused. To enable this, you need the *web session ID*, which you save in the Workflow-based variables ahead of time.
+=== Reuse Web Session 
+image:toolbox-variable-web-automation-web-session-images/image1.png[50%, 50%, An image of the Reuse web session section of the wizard]
 
-image:toolbox-variable-web-automation-web-session-images/image2.png[An image showing the web session ID]
+The *Reuse Web Session* section defines which web session to reuse. To reuse a web session, use the *web session ID* that you previously saved in the Workflow-based variables.
 
-You can define a name and a description and the value is assigned
-automatically. Once you have created the web session ID, you can link it
-with the session easily, using the drop-down in the Web Session Wizard.
+image:toolbox-variable-web-automation-web-session-images/image2.png[75%, 75%, An image showing the web session ID]
 
-image:toolbox-variable-web-automation-web-session-images/image3.png[An image showing the reuse web session option]
+You can define a name and a description and then RPA Builder assigns a value automatically. Once you have created the web session ID, you can link it
+with the session using the drop-down in the Web Session Wizard.
+
+image:toolbox-variable-web-automation-web-session-images/image3.png[50%, 50%, An image showing the reuse web session option]
```

Comment:
Though it is an alt text, it would be great to add the full stop at the end of the sentence to improve the pronunciation of the speech machine.